SummerSchool/.idea/assetWizardSettings.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
by default i would not have anything from the .idea folder checked in.
i understand code formatting conventions.
but more than that? 
are you sure?
is this a practice i'm not aware of ?

Reply:
I am sure that I added the entire folder of .idea into .gitignore, but somehow it always pushed those files. The all files in this folder is generated automatically by IntelliJ.